.github/workflows/vulnerabilities.yml

```diff
@@ -0,0 +1,25 @@
+name: Check CVE
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 * * * *'
+
+jobs:
+  check-vulnerabilities:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Update local CVE database
+        run: ./gradlew dependencyCheckUpdate
+      - name: Run Vulnerability Check
+        run: ./gradlew dependencyCheckAnalyze
+      - name: Filter Top-Level Vulnerabilities
+        if: ${{ always() }}
```

Comment:
check possibility for [continue-on-error](https://docs.github.com/de/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error)